app/stats.go

```diff
@@ -5,10 +5,10 @@ import (
 	"github.com/codegangsta/cli"
 )
 
-func LsStats() cli.Command {
+func LsJournal() cli.Command {
 	return cli.Command{
-		Name:      "ls-stats",
-		ShortName: "stats",
+		Name:      "ls-journal",
```

Comment:
`ls-journal` is simply to match similar commands like `ls-replicas`. It's likely to be the only command for a while. But who knows?.. 
If you think it makes it's not helping than I can remove it?